packages/config/src/main.js

```diff
@@ -85,13 +84,9 @@ const resolveConfig = async function (opts) {
     featureFlags,
   })
 
-  const { base: baseA, config: configA } = await addBase(repositoryRoot, config)
-  const buildDir = await getBuildDir({ config: configA, repositoryRoot, base: baseA, logs, featureFlags })
-  const configB = await resolveConfigPaths({ config: configA, repositoryRoot, buildDir, baseRelDir: baseRelDirA })
```

Comment:
This PR purpose is mostly about moving those lines from the end of `@netlify/config` to a little earlier in the code logic.